core/services/versionchooser/main.py

```diff
@@ -43,11 +43,18 @@ async def get_available_versions(repository: str, image: str) -> Any:
     return await versionChooser.get_available_versions(f"{repository}/{image}")
 
 
+async def load(request: web.Request) -> Any:
```

Comment:
our images are at ~500mb, right?
I'm concerned about a possible OOM error down the road on the 1GB boards.
This would be fixed with the third snippet [here](https://docs.aiohttp.org/en/v3.8.0/web_quickstart.html#file-uploads).
We could file an issue to fix this later, up to you.

Reply:
We can open an issue about that later. I got some problems working with forms inside aiohttp and openapi schema.